lib/screens/homepage/blog_body.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+
+class MyBlogBody extends StatefulWidget {
+  const MyBlogBody({Key? key}) : super(key: key);
+
+  @override
+  State<MyBlogBody> createState() => _MyBlogBodyState();
+}
+
+class _MyBlogBodyState extends State<MyBlogBody> {
+  String _blogText = 'Welcome to my blog';
+  final String _indiaBlogText = 'This is the country India. This is a beautiful country. It is a secular country, not a developed country.';
+  final String _canadaBlogText = 'This is the country Canada. This is not USA. It is a separate country by itself.';
+  final String _chinaBlogText = 'This is the country China. It is very different from other countries of the world. It manages its own matters by itself.';
```

Comment:
Why are they not `const`?

Reply:
They should be `static const`. Refactored them.